addons/mail/views/discuss_channel_views.xml

```diff
@@ -28,6 +28,7 @@
                                     <img t-att-src="kanban_image('discuss.channel', 'avatar_128', record.id.raw_value)" class="oe_module_icon" alt="Channel"/>
                                     <div class="oe_module_desc">
                                         <h4 class="o_kanban_record_title">#<field name="name"/></h4>
+                                        <field name="channel_type" groups="base.group_no_one"/>
```

Comment:
Extra fields in debug are not necessary for the feature to work (obviously) but helped a lot when I debugged it so I'd rather keep them in.